Assets/Scripts/Ships/DataManagement/ShipData.cs

```diff
@@ -19,7 +19,10 @@ public class ShipData : UniqueId
         [SerializeField] private GameObject visuals;
         [SerializeField] private int cost;
         [SerializeField] private List<AttackScriptableObject> weapons;
-        [SerializeField] private List<ModuleList> moduleGrid;
+        [SerializeField] private int moduleGridHeight;
+        [SerializeField] private int moduleGridWidth;
+        [SerializeField] private List<Module> moduleList;
+        [SerializeField] private List<ModuleSlot> moduleSlotList;
```

Comment:
Think it's worth thinking about a better way to store design time module grid data (possible two SOs, one for describing the grid/slots & one for default modules)